insights/core/remote_resource.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import requests_cache
+
+
+class RemoteResource(object):
+    """
+    RemoteResource class for accessing external Web resources.
+
+    Functions:
+        get(): Calls `get()` function of the RemoteRersource superclass and returns the response payload.
+
+    Each line is parsed into a dictionary with the following keys:
```

Comment:
@lhuett did you mean to add some doc here?

Reply:
Cleaned this up